p4rt_app/utils/authz_logger_test.cc

```diff
@@ -125,10 +125,10 @@ TEST_F(AuthzLoggerTest, PermittedRpcIsLoggedInDb) {
 
   ASSERT_TRUE(WaitUntil(
```

Comment:
Based on the other PR comments, the authz & metric_recorder related file changes may not be required. Can you move the files above to a different PR ?